### PR TITLE
Fall back to a JavaTimer if a DefaultTimer instance can't be fetched

### PR DESCRIPTION
**Pull request by samstarling**
@Fluxx: This supersedes #28. Take a look and let me know what you think, and I'll try and get this merged/released ASAP so that you can validate whether it helps or not. Hopefully it will.

### CODE REVIEW

File: src/main/scala/com/samstarling/prometheusfinagle/PrometheusStatsReceiver.scala [hunk 1]
```diff
@@ -7,12 +7,16 @@ import scala.collection.concurrent.TrieMap
 import com.twitter.finagle.util.DefaultTimer
 import com.twitter.util._
 
+private object CustomDefaultTimer {
+  def getInstance: Timer = Option(DefaultTimer.getInstance).getOrElse(new JavaTimer())
```
Comment (Fluxx) [inline src/main/scala/com/samstarling/prometheusfinagle/PrometheusStatsReceiver.scala]: Might want to make this `daemon=true`? That's [what Finagle does](https://github.com/twitter/finagle/blob/develop/finagle-core/src/main/scala/com/twitter/finagle/util/DefaultTimer.scala#L84).

**Comment by Fluxx** [comment]
Generally looks good to me though, sans the `daemon` question.

**Comment by samstarling** [comment]
@Fluxx Good call. Once this has built, I'll merge and release. Sorry it's taken a while!

**Review by Fluxx — APPROVED** [review]
Looks good to me now, though the build seems to be failing.

**Comment by Fluxx** [comment]
Just checking in on this.

**Comment by samstarling** [comment]
@Fluxx Sorry for the delay: I've just released `0.0.10` to Bintray, and it's syncing to Maven Central at the moment. Can you let me know if it fixes things for you? I still think there's a race condition in there somewhere, but I'll pick that up as a separate issue.

**Comment by Fluxx** [comment]
I'll give it a shot and see. I had actually internally been using the old version but with an alternate constructor usage to use the `JavaTimer`. However, I ran into another issue involving any stats which don't seem to have any amount of nesting. I believe the metric is coming from finagle itself.

**Pull request merged**